core/importer/datasets_exporter.py

```diff
@@ -0,0 +1,51 @@
+import json
+import sys
+
+from core.models import  Dataset
+from core.utils import DaisyLogger
+from django.conf import settings
+from io import StringIO
+
+logger = DaisyLogger(__name__)
+
+
+class DatasetsExporter:
+
+    def export_to_file(self, file_handle, stop_on_error=False, verbose=False):
+        result = True
+        try:
+            buffer = self.export_to_buffer(StringIO())
+            print(buffer.getvalue(), file=file_handle)
+        except Exception as e:
+            logger.error('Dataset export failed')
+            logger.error(str(e))
+            result = False
+        logger.info('Dataset export complete see file: {}'.format(file_handle))
+        return result
+
+
+
+    def export_to_buffer(self, buffer, stop_on_error=False, verbose=False):
+        dataset_dicts = []
+        datasets = Dataset.objects.all()
+        for dataset in datasets:
+            logger.debug(' * Exporting dataset: "{}"...'.format(dataset.__str__()))
```

Comment:
minor comment: you can do `see file: {}'.format(dataset)`
even better: `logger.debug(' * Exporting dataset: "%s"...', dataset)` this way, the str method and the concatenation are not executed if not in debug mode